From 2f1feb1efd6f2026232e0daae231d85cd4bff9fe Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Fri, 28 Dec 2012 11:09:50 +0800
Subject: [PATCH 133/304] ENGR00238425 fix libvpu build error issue

The gpu make file must be included after
the BoardConfigure common file.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 sabresd_6dq/BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 54172e3..3d0786f 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -4,8 +4,8 @@
 
 include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/sabresd_6dq/build_id.mk
-include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 include device/fsl/imx6/BoardConfigCommon.mk
+include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
 TARGET_BOOTLOADER_BOARD_NAME := SABRESD
 PRODUCT_MODEL := SABRESD-MX6DQ
-- 
1.8.0

